glide.lock

```diff
@@ -223,7 +223,7 @@ imports:
   - transport
 - name: gopkg.in/src-d/go-errors.v0
   version: 0c303ec4c027302259ec1738f19f515aad25f13f
-- name: gopkg.in/src-d/simple-linguist.v1
+- name: gopkg.in/src-d/enry.v1
```

Comment:
To be honest, at this point I don't know what we should or shouldn't be doing with respect to versioning, is there any document or meeting minute detailing this?